docs/release-notes/1.8.0.asciidoc

```diff
@@ -0,0 +1,43 @@
+:issue: https://github.com/elastic/cloud-on-k8s/issues/
+:pull: https://github.com/elastic/cloud-on-k8s/pull/
+
+[[release-notes-1.8.0]]
+== {n} version 1.8.0
+
+[[breaking-1.8.0]]
+[float]
+=== Breaking changes
+
+* Remove non-functional pre-stop hook parts {pull}4801[#4801] (issue: {issue}4698[#4698])
+
+The operator does not rely on the `$PRE_STOP_MAX_WAIT_SECONDS` environment variable to control <<{p}-prestop,Elasticsearch Pods PreStop hook>> anymore. Use `$PRE_STOP_ADDITIONAL_WAIT_SECONDS` instead (defaults to 50 seconds).
+
+* Note the next minor ECK version (1.9.0) will drop support for Openshift 3.11.
+
+[[enhancement-1.8.0]]
+[float]
+=== Enhancements
+
+* Drop "properties: {}" from empty-v1alpha1-patch {pull}4813[#4813]
+* Remove beta label from Helm documentation {pull}4802[#4802] (issue: {issue}4796[#4796])
+
+[[bug-1.8.0]]
+[float]
+=== Bug fixes
+
+* Fix handling CAs in Agent controller {pull}4807[#4807] (issue: {issue}4790[#4790])
+* Improve Elasticsearch cluster health API call when timeout {pull}4803[#4803] (issue: {issue}4507[#4507])
```

Comment:
Feels like this is only half of a sentence

Reply:
👍 Changing to "Improve Elasticsearch cluster health API timeout handling"